viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -863,4 +879,8 @@ export class ComboControls extends EventDispatcher {
     _panVector.multiplyScalar(distance);
     _targetEnd.add(_panVector);
   };
+
+  private isIdentityQuaternion(q: THREE.Quaternion) {
+    return q.x === 0 && q.y === 0 && q.z === 0 && q.w === 1;
+  }
```

Comment:
Cant this be used instead? https://threejs.org/docs/#api/en/math/Quaternion.identity

Reply:
This is a method to make some quaternion an identity, not to check whether it is equal to identity.